Pipfile

```diff
@@ -11,6 +11,8 @@ pre-commit = "*"
 coverage = "*"
 ipython = "*"
 pandas = "*"
+faker = "*"
+factory-boy = "*"
```

Comment:
[Factory Boy](https://factoryboy.readthedocs.io/en/latest/) is a library for making fake versions of your models with randomized data, which makes tests easier to write. Based on Factory Girl from Ruby, because programmers are lazy when it comes to naming